### PR TITLE
add federatedETL.federator.primaryClusterID

### PR DESCRIPTION
**Pull request by jessegoodier**
Add `--set federatedETL.federator.primaryClusterID=$CLUSTER_NAME` per 1.101 update
Add backup considerations

### CODE REVIEW

**Review by bstuder99 — DISMISSED** [review]

File: federated-etl.md
Comment (thomasvn) [inline federated-etl.md:13]: Perhaps adjust this?

> Many solutions that aggregate Prometheus metrics (like Thanos) often to not scale well in large environments

to

> Many solutions that aggregate Prometheus metrics (like Thanos), are often expensive to scale in large environments

**Comment by thomasvn** [comment]
@jessegoodier I'm in the middle of working on a new Federated ETL Backups & Alerting doc. I'll try to have a PR out by end of day.

File: federated-etl.md
Comment (jessegoodier) [reply federated-etl.md:12]: As long as the metrics we need are available, this method is more efficient. 

Determining if metrics are missing should be easier. See this: https://github.com/kubecost/cost-analyzer-helm-chart/issues/2066

Comment (thomasvn) [reply federated-etl.md:12]: Thanks!

File: federated-etl.md
Comment (jessegoodier) [reply federated-etl.md:13]: much better. will commit. 

Any other changes based on the work you are doing? I'll wait for your approval before committing. 

Not urgent.

**Comment by jessegoodier** [comment]
@thomasvn updated, please review.
I put the link at bottom as this topic is chronologically later in a setup.

File: federated-etl.md
Comment (thomasvn) [inline federated-etl.md:120]: Can we adjust the link here to instead show

```
[backups and alerting](/federated-etl-backups-alerting.md)
```

This way, if you're reading the doc on Github you get redirected within Github. If you're reading on Gitbook, you get redirected within Gitbook

**Comment by bstuder99** [comment]
Thomas, I'm fine with that change generally but when I try to do that here, it 404s with https://github.com/kubecost/docs/blob/fed-etl-updates/federated-etl-backups-alerting.md. I'm hoping when we merge the branch (where it thinks this doc will be located), that problem will resolve itself. If not, I can correct the error post-merge.

**Review by bstuder99 — APPROVED** [review]

**Comment by thomasvn** [comment]
@bstuder99 Great! I'm good with that!

**Pull request merged**
